python/pylibcudf/pylibcudf/libcudf/nvtext/stemmer.pxd

```diff
@@ -1,16 +1,17 @@
 # Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
 from libc.stdint cimport int32_t
+from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
 from pylibcudf.libcudf.column.column cimport column
 from pylibcudf.libcudf.column.column_view cimport column_view
 from pylibcudf.libcudf.types cimport size_type
 
 
 cdef extern from "nvtext/stemmer.hpp" namespace "nvtext" nogil:
-    ctypedef enum letter_type:
-        CONSONANT 'nvtext::letter_type::CONSONANT'
-        VOWEL 'nvtext::letter_type::VOWEL'
+    cpdef enum class letter_type(bool):
```

Comment:
There isn't a storage class (`bool`) defined for this enum class in the C++ code... we should probably define that at the C++ level, so it looks like this:

https://github.com/rapidsai/cudf/blob/02ee8199a7703eafe8aa061213423eecf27757ec/cpp/include/cudf/types.hpp#L118-L121

Reply:
I wonder if I can just get rid of the `bool` instead

Reply:
Yes, I think that shouldn't have been there, given the enum class is defined without it

Reply:
Seems fine to me.